app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -130,20 +130,24 @@ class HomeFragment : Fragment() {
     private val args by navArgs<HomeFragmentArgs>()
     private lateinit var bundleArgs: Bundle
 
+    private var _binding: FragmentHomeBinding? = null
```

Comment:
I think we need something like : 
```
override fun onDestroy(){
..
_binding = null
}
```
right?

Reply:
Right! Thanks!